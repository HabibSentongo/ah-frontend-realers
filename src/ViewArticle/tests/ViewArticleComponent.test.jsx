import React from 'react';
import { shallow } from 'enzyme';
import ViewArticleComponent from '../components/VeiwArticleComponent';


describe('ViewArticleComponent', () => {
  const props = {
    moment: jest.fn(),
    singleArticle: {
      title: 'Logged in',
      description: 'You can close this page and return to your CLI. It should now be logged in.',
      body: 'You can close this page and return to your CLI. It should now be logged in.',
      createdAt: '2019-07-22T05:19:22.188734Z',
      updatedAt: '2019-07-22T05:19:22.188788Z',
      author: {
        email: 'alexxsanya@gmail.com',
        username: 'alexxsanya',
        bio: '',
        image: '',
        gender: 'male',
        first_name: 'Alex',
        last_name: 'Sanya',
        location: '',
        birth_date: null,
      },
      slug: 'logged-in',
      image: 'https://firebasestorage.googleapis.com/v0/b/ah-frontend-realers.appspot.com/o/article-image%2Fheroku.png?alt=media&token=f9f6d723-c7d4-4df3-b4de-d1e84a5222bf',
      average_rating: 0,
      tagName: [
        'heroku',
      ],
    },
  };
  it('render the viewArticle component', () => {
    const wrapper = shallow(<ViewArticleComponent {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
