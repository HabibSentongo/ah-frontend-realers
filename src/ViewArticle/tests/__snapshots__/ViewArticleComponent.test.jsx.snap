// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewArticleComponent render the viewArticle component 1`] = `ShallowWrapper {}`;
